docs/standards/generic-standards/02-lsp1-universal-receiver.md

```diff
@@ -37,7 +37,15 @@ An **optional extension** to this standard is proposed to let contracts not just
 
 This is possible through setting the address of the **external contract** as value for the **LSP1UniversalReceiver key** below, in the **[ERC725Y key value store](https://github.com/ERC725Alliance/erc725/blob/main/docs/ERC-725.md#erc725y)** of the contract implementing the **universalReceiver** function. 
 
-![LSP1UniversalReceiverKey](../../../static/img/lsp1-key.png)
```

Comment:
If this file is not being used anymore can you also delete it from the repo?

Reply:
Done